zstacklib/zstacklib/utils/http.py

```
@@ -303,7 +303,16 @@ def _build(self):
         cherrypy.engine.autoreload.unsubscribe()
         site_config = {}
-        site_config['server.socket_host'] = '0.0.0.0'
+        # IPv6 双栈监听：'::' 在 Linux(IPV6_V6ONLY=0)下同时接受 IPv4/IPv6 连接
+        # 若系统禁用 IPv6（disable_ipv6=1），自动回退到 0.0.0.0
+        try:
+            import socket as _socket
+            _s = _socket.socket(_socket.AF_INET6, _socket.SOCK_STREAM)
+            _s.close()
+            bind_ip = '::'
+        except OSError:
+            bind_ip = '0.0.0.0'
```

Ruff (0.15.12)

[error] Possible binding to all interfaces

(S104)